src/components/views/rooms/RoomListHeader.tsx

```diff
@@ -367,7 +373,7 @@ const RoomListHeader: React.FC<IProps> = ({ onVisibilityChange }) => {
 
     let title: string;
     if (activeSpace) {
-        title = spaceName;
+        title = spaceName!;
```

Comment:
In cases like these should we just check the variable to be set?
Here for instance

```
if (activeSpace && spaceName) {
```

Reply:
Perhaps, this is a case where we know if activeSpace is set that we also set spaceName, TS doesn't really have a sane way of expressing these relationships